local-volume/bootstrapper/README.md

```diff
@@ -50,20 +50,17 @@ metadata:
   name: local-volume-config
   namespace: kube-system
 data:
-  "local-fast": |
-    {
-      "hostDir": "/mnt/ssds",
-      "mountDir": "/local-ssds"
-    }
-  "local-slow": |
-    {
-      "hostDir": "/mnt/hdds",
-      "mountDir": "/local-hdds"
-    }
-  "local-storage": |
-    {
-      "hostDir": "/mnt/disks"
-    }
+  someOtherParam: |
```

Comment:
Remove someOtherParam because it's not a real config parameter we support.

Reply:
done